lib/realtime/timepoint_status.ex

```diff
@@ -179,7 +175,8 @@ defmodule Realtime.TimepointStatus do
               run_id: trip.run_id,
               time_since_trip_start_time: now_time_of_day - trip.start_time,
               headsign: trip.headsign,
-              via_variant: RoutePattern.via_variant(trip.route_pattern_id),
+              via_variant:
+                trip.route_pattern_id && RoutePattern.via_variant(trip.route_pattern_id),
```

Comment:
@skyqrose this would appear to be a separate fix, is that correct?

Reply:
Yeah. It just popped up with the changed test data.